java/java-ranger-regression/WBS_prop1/Main.java

```diff
@@ -0,0 +1,305 @@
+import org.sosy_lab.sv_benchmarks.Verifier;
+public class Main {
+	
+	//Internal state
+	private int WBS_Node_WBS_BSCU_SystemModeSelCmd_rlt_PRE; 
+	private int WBS_Node_WBS_BSCU_rlt_PRE1; 
+	private int WBS_Node_WBS_rlt_PRE2;
+
+	//Outputs
+	private int Nor_Pressure;
+	private int Alt_Pressure;
+	private int Sys_Mode;
+
+	public Main() {
+		WBS_Node_WBS_BSCU_SystemModeSelCmd_rlt_PRE = 0;
+		WBS_Node_WBS_BSCU_rlt_PRE1 = 0;
+		WBS_Node_WBS_rlt_PRE2 = 100;
+		Nor_Pressure = 0;
+		Alt_Pressure = 0;
+		Sys_Mode = 0;
+	}
+
+	public void update(int PedalPos, boolean AutoBrake,
+			boolean Skid) {
+		int WBS_Node_WBS_AS_MeterValve_Switch; //4
+		int WBS_Node_WBS_AccumulatorValve_Switch; //5
+		int WBS_Node_WBS_BSCU_Command_AntiSkidCommand_Normal_Switch; //6
+		boolean WBS_Node_WBS_BSCU_Command_Is_Normal_Relational_Operator; //7
+		int WBS_Node_WBS_BSCU_Command_PedalCommand_Switch1; //8
+		int WBS_Node_WBS_BSCU_Command_Switch; //9
+		boolean WBS_Node_WBS_BSCU_SystemModeSelCmd_Logical_Operator6; //10
+		int WBS_Node_WBS_BSCU_SystemModeSelCmd_Unit_Delay; //11
+		int WBS_Node_WBS_BSCU_Switch2; //12
+		int WBS_Node_WBS_BSCU_Switch3; //13
+		int WBS_Node_WBS_BSCU_Unit_Delay1; //14
+		int WBS_Node_WBS_Green_Pump_IsolationValve_Switch; //15
+		int WBS_Node_WBS_SelectorValve_Switch; //16
+		int WBS_Node_WBS_SelectorValve_Switch1; //17
+		int WBS_Node_WBS_Unit_Delay2; //18
+		
+	   WBS_Node_WBS_Unit_Delay2 = WBS_Node_WBS_rlt_PRE2; 
+	   WBS_Node_WBS_BSCU_Unit_Delay1 = WBS_Node_WBS_BSCU_rlt_PRE1; 
+	   WBS_Node_WBS_BSCU_SystemModeSelCmd_Unit_Delay = WBS_Node_WBS_BSCU_SystemModeSelCmd_rlt_PRE; 
+
+	   WBS_Node_WBS_BSCU_Command_Is_Normal_Relational_Operator = (WBS_Node_WBS_BSCU_SystemModeSelCmd_Unit_Delay == 0); 
+
+	   if ((PedalPos == 0)) {
```

Comment:
I personally think, it might be interesting to distribute the very same benchmark file with a switch-case statement instead of this if-then-else cascade in order to integrate some more features of the Java language. From my point of view, this seems to be the ideal candidate to use this test case in two descriptions.

Reply:
I agree. I will add this test case too into the repo soon. I've currently also got quite a few other benchmarks to add and will add this into my list.